app/database/models/usersModel.js

```diff
@@ -0,0 +1,31 @@
+const db = require('../dbConfig');
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/cse110-fa21-group17/cse110-fa21-group17/pullRequest?prid=8408698)
```suggestion
const db = require("../dbConfig");
```

Reply:
Not sure what Codacy is doing here... Maybe it's using a style guide I have not used before, feel like this should be a group discussion on which style guide to use. I was following JS style guide: 
https://google.github.io/styleguide/jsguide.html#features-string-literals

Reply:
Don't worry about this too much now as we are setting up the CI/CD pipeline using git action. We are still testing things out. Ignore this auto generated style suggestions for now.

Reply:
Do you guys still need the help? I just came back